### DISCUSSION TITLE
一个自动添加最优 IP 到 Adguard Home 的工具

### DISCUSSION THREAD

**Discussion by a270443177** [XIU2/CloudflareSpeedTest#507] [分享]
项目地址:  https://github.com/a270443177/CloudflareSpeedTest-Adguard-Script

将脚本以及配置文件放在CloudflareSpeedTest，同级目录下，直接执行次项目脚本会自动将解析上传、更新到AdGuardHome中。

![Snipaste_2024-03-21_16-45-23](https://github.com/XIU2/CloudflareSpeedTest/assets/6148389/a9db5f7d-bf01-45a2-9bff-9a66547db2b5)

**Comment by yyyytawa** [comment]
其实你可以用这个来实现

||www.cloudflare.com^$dnsrewrite=cfip.yyyyt.top

cfip.yyyyt.top 换成你的优选域名即可
只需要一个脚本自动更新cfip.yyyyt.top的解析结果就行
